client/src/hooks/useCCDLDataset.js

```diff
@@ -0,0 +1,9 @@
+import { useContext } from 'react'
+import { CCDLDatasetContext } from 'contexts/CCDLDatasetContext'
+
+export const useCCDLDataset = () => {
+  const { datasets } = useContext(CCDLDatasetContext)
+  return { datasets }
```

Comment:
It might be helpful to expose `setQuery` from the hook too - that way, components could update the `query` more easily based on user selections. Not sure if that’s the plan, but thought I’d mention it!

```suggestion
  const { datasets, setQuery } = useContext(CCDLDatasetContext)
  return { datasets, setQuery }
```